packages/api/tests/endpoints/collections/delete-collection.js

```diff
@@ -184,9 +219,24 @@ test('Deleting a collection removes it from all data stores', async (t) => {
       constructCollectionId(originalCollection.name, originalCollection.version)
     )
   );
+
+  const { Messages } = await sqs().receiveMessage({
+    QueueUrl: t.context.QueueUrl,
+    WaitTimeSeconds: 10,
+  }).promise();
+
+  t.is(Messages.length, 1);
+
+  const message = JSON.parse(JSON.parse(Messages[0].Body).Message);
+  t.is(message.event, 'Delete');
+  t.truthy(message.deletedAt);
```

Comment:
```suggestion
  t.true(Date.now() > message.deletedAt);
```

Slightly prefer this because I think it demonstrates that `message.deletedAt` is a timestamp

Reply:
[`754f871` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/754f871e6b2a2231b24c1dc75a24f09bf623fabd)